ai-ml/aibrix/examples/deepseek-distill.yaml

```diff
@@ -0,0 +1,95 @@
+apiVersion: v1
```

Comment:
This file should be moved under https://github.com/awslabs/data-on-eks/tree/main/gen-ai/inference